src/openepd/api/utils.py

```diff
@@ -26,3 +26,18 @@ def encode_path_param(value: str) -> str:
     :return: encoded value
     """
     return quote(value, safe="")
+
+
+def remove_none_id_fields(d: dict) -> dict:
+    """Remove any key 'id' with a None value from the dictionary, including nested dicts."""
```

Comment:
Describe args and return type, say something about absence of side effects.

Reply:
updated